templates/components/checkout-steps.tpl

```diff
@@ -82,7 +82,7 @@
         </p>
       </div>
 
-      <div class="{$componentName}__step" data-step="2">
+      <div class="{$componentName}__step" data-step="checkout-addresses-step">
```

Comment:
@NeOMakinG Is this correct? The d-none thing on steps

Reply:
The d-none? Steps are hidden if it's not the current and it's toggled using JS if you switch to another one

Reply:
Yes, but why is specifically the second one displayed by default?

Reply:
Did hide every steps by default now